lib/matplotlib/collections.py

```diff
@@ -475,7 +475,7 @@ def set_offsets(self, offsets):
 
         Parameters
         ----------
-        offsets : float or sequence of floats
+        offsets : array-like (N, 2) or (2, )
```

Comment:
```suggestion
        offsets : array-like (N, 2) or (2,)
```
At least, that's the `repr()` format of a 1-element tuple. Also looks better IMHO.